starlingcaptureapi/config.py

```diff
@@ -25,6 +25,8 @@
 # API token for the web3.storage service
 WEB3_STORAGE_API_TOKEN = os.environ.get("WEB3_STORAGE_API_TOKEN")
 
+# Full path to Opentimestamps client. Must be already configured.
+OTS_CLIENT_PATH = os.environ.get("OTS_CLIENT_PATH")
```

Comment:
Let's try to keep these alphabetized in the file -- makes them easier to find when scanning through things later.

Reply:
None of the other entries are alphabetized, maybe this file and the `.env.example` can be cleaned up in another PR?